rad/monadic/issue-remote.rad

```diff
@@ -138,6 +209,28 @@
      (fn [cs] (add-right c_ cs)))
     (mark-used-nonce (lookup :nonce c))))
 
+(def merge-new
+  (fn [new]
+    (fn [old]
+      (<> old new))))
+
+(def edit-issue
+  "Modify some of the attributes of an issue."
+  (fn [i]
+    (validator/issue-edit-input i)
+    (def i_ (delete :author (strip-input i)))
```

Comment:
Isn't `:author` guaranteed to be the same? Why delete it?

Reply:
No, for the moment anyone can edit an issue.

Reply:
Ok, so a *comment* edit must be by the same person, but not an issue edit?  I guess then the Q is just valid below.

Reply:
Yes, I thought this was consistent with GitHub, but, as you will notice with your comment above, that is not the case :open_mouth:

Reply:
FWIW, I'm happy with comments being editable or not.